data/net/minecraft/world/level/validation/DirectoryValidator.mapping

```diff
@@ -3,16 +3,17 @@ CLASS net/minecraft/world/level/validation/DirectoryValidator
 		ARG 1 symlinkTargetAllowList
 	METHOD validateSave (Ljava/nio/file/Path;Z)Ljava/util/List;
 		ARG 1 path
+		ARG 2 forceAllowSymlink
 	METHOD validateSymlink (Ljava/nio/file/Path;Ljava/util/List;)V
 		ARG 1 directory
 		ARG 2 entries
 	CLASS 1
 		METHOD preVisitDirectory (Ljava/nio/file/Path;Ljava/nio/file/attribute/BasicFileAttributes;)Ljava/nio/file/FileVisitResult;
-			ARG 1 path
-			ARG 2 attributes
+			ARG 1 dir
+			ARG 2 attrs
 		METHOD validateSymlink (Ljava/nio/file/Path;Ljava/nio/file/attribute/BasicFileAttributes;)V
 			ARG 1 path
-			ARG 2 attributes
+			ARG 2 attrs
 		METHOD visitFile (Ljava/nio/file/Path;Ljava/nio/file/attribute/BasicFileAttributes;)Ljava/nio/file/FileVisitResult;
-			ARG 1 path
-			ARG 2 attributes
+			ARG 1 file
+			ARG 2 attrs
```

Comment:
@ other reviewers: these parameters are copied from `java.nio.file.SimpleFileVisitor`. Personally, I think the existing ones are better (i.e. `attributes` over `attrs`) and more consistent with our standard. But it is different from the super-class parameters.

My vote is to keep `path`, `attributes`, etc.

Reply:
While keeping `attributes` over `attrs` is fine, I think `file` and `dir` are better and convey more information. A Path could be either, but the Path passed as an argument will be a file or directory afaik. I then think its better to keep the specific `file` and `dir` names.

Reply:
I agree with MachineMaker on this. I prefer `attributes` over `attrs`, but think `file` and `dir` are better.

Reply:
2 Cents:
attributes -> attr  | For readability
file & dir               | For readability

Question is if we should plan a sit down to make a pool of standardized names that we then apply over the entire project like we did before with tileEntity -> blockEntity and such?

Reply:
I'm ambivalent to `file` / `dir`. My main comment is, in keeping with Parchment standards, to avoid abbreviations or shortenings (like `attrs` or `dir`) instead of the actual word (`attributes`, `directory`).

It's not a matter of having a standardized naming pass; it's a matter of sticking to the existing standards.

> 4. Avoid abbreviations or acronyms.

Reply:
The thing is also while yes "dir" is an abbreviation, it's such a established term in programming as a whole that I'm not sure I'd consider it a "pure abbreviation" rather than an established contextual word.

I can honestly go either way with dir vs directory so /shrug

Reply:
So currently it's using `attributes` and `path` and `dir`.